docs/Guide/application-commands/application-command-registry/advanced/setting-global-behavior-when-not-identical.mdx

```diff
@@ -3,4 +3,12 @@ sidebar_position: 3
 title: Configuring the global behavior when not identical
 ---
 
-<!-- TODO: https://vladfrangu.notion.site/Application-Command-Registry-e1be0c1022bc429f8aab6fb92cd3915d -->
+Configuring the global behavior is easy, just import the [`ApplicationCommandRegistry`][application-command-registry] class from `@sapphire/framework` and set it to your desired option. For example, to override the behavior to overwrite by default, you may write:
```

Comment:
Changing this as we'll default to overwrite soon

```suggestion
Configuring the global behavior is easy! You just need to import the [`ApplicationCommandRegistries`][application-command-registries] object from `@sapphire/framework` and call the `setDefaultBehaviorWhenNotIdentical` method with the new desired default behavior. For example, to override the behavior to only log differences by default, you may write:
```

Reply:
I thougth so, wasn't totally sure yet. Will note that 👍